### ISSUE TITLE
Wrong config to Symfony 4.3

### ISSUE THREAD

**Issue by Big-Shark** [theofidry/PsyshBundle#43]
I received this error message
```
TypeError: Argument 1 passed to Symfony\Bundle\FrameworkBundle\Test\WebTestCase::getClient() must be an instance of Symfony\Bundle\FrameworkBundle\KernelBrowser or null, instance of Symfony\Bundle\FrameworkBundle\Client given, called in /app/vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Test/WebTestCase.php on line 55
```
And I found a problem we receive 'Symfony\Bundle\FrameworkBundle\Client', but we need KernelBrowser
```
private static function getClient(KernelBrowser $newClient = null): ?KernelBrowser
```
And Symfony [points to](
https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Resources/config/test.xml#L14) KernelBrowser
But you [points to](https://github.com/theofidry/PsyshBundle/blob/master/resources/config/test.xml#L14) Client

I remove you test.client in my project, and now all works good, please, check this.

**Comment by kubk** [comment]
Thanks for digging into this.

It looks like this bundle is currently uncompatible with Symfony 4.3 due to changes in TestContainer: https://github.com/symfony/symfony/commit/6764d4e01272b922f6f3c9ab2e3dee2fd19acd43#diff-ca33e3189d79b860cb883635e1f11c36
Moreover, this class is marked as _internal_ now. I believe we still can benefit from this class. For example, 'test.service_container' allows to use `Doctrine\ORM\EntityManagerInterface::class` in Psysh.
The simplest solution would be to replace current resources/config/test.xml with [actual version](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Bundle/FrameworkBundle/Resources/config/test.xml) from master and require Symfony 4.3 as minimal supported version in composer.json. It's a BC break.

@theofidry What do you think, do you have any suggestions? I can create a PR with an updated xml config to fix the issue.

**Comment by theofidry** [comment]
@kubk I'm fine with requiring 4.3 as a minimum version

**Comment by theofidry** [comment]
Closed by #44
